src/containers/Playlist.js

```diff
@@ -45,46 +45,51 @@ class Playlist extends Component {
     return false
   }
 
-  makeSongRejectList = (rejects) => {
+  makeSongRejectList = rejects => {
     return rejects.map(item => {
-      return <SongItemReject message={item.macarenaURLValidity.message} key={item.id} song={item} />
+      return (
+        <SongItemReject
+          message={item.macarenaURLValidity.message}
+          key={item.id}
+          song={item}
+        />
+      )
     })
   }
 
-
-  render () {
+  render() {
     const {
       currentOpenPlaylist,
       isCurrentPlaylistLoaded,
       handlePlaylistSelect,
-      playlistSortObj,
-      currentRoute,
       currentOpenPlaylistRejects,
       toggleShowRejects,
-      showRejects,
+      showRejects
     } = this.props
 
     if (isCurrentPlaylistLoaded && currentOpenPlaylist) {
-      const renderList = this.makeSongList(currentOpenPlaylist.contents, handlePlaylistSelect)
+      const renderList = this.makeSongList(
+        currentOpenPlaylist.contents,
+        handlePlaylistSelect
+      )
       const rejectCount = currentOpenPlaylistRejects.length
       return (
-        <div className='w-100 min-vh-100'>
-          {
-            renderList
-          }
-          {
-            showRejects
-            ? this.makeSongRejectList(currentOpenPlaylistRejects)
-            : <div />
-          }
-          {
-            rejectCount > 0
-            ? <ToggleRejectedSongs
+        <div className="w-100 min-vh-100">
+          {renderList}
+          {showRejects ? (
+            this.makeSongRejectList(currentOpenPlaylistRejects)
+          ) : (
+            <div />
+          )}
+          {rejectCount > 0 ? (
```

Comment:
I dunno how I feel about this style of ternary. If there is as much extra syntax then IMO we should just use if/else

Reply:
def a nitpick though